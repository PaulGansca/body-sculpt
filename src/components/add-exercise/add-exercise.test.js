import { shallow } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import AddExercise from './add-exercise';

let wrapper;

beforeEach(() => {
    const mockProps = {
        getState: jest.fn(),
        subscribe: jest.fn(),
        dispatch: jest.fn(),
    }
    wrapper = shallow(
                <Provider store={mockProps}>
                    <AddExercise />
                </Provider>
                );
})

describe('<AddExercise />', () => {
  it("expect to render add exercise modal", () => {
    expect(wrapper).toMatchSnapshot();
  })
});