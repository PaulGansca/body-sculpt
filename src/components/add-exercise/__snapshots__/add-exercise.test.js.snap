// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddExercise /> expect to render add exercise modal 1`] = `ShallowWrapper {}`;
